### PR TITLE
Added documentation for Celery Beat auto monitoring

### PR DESCRIPTION
**Pull request by antonpirker**
We have a new feature that allows Celery Beat periodic tasks to be automatically monitored with Sentry Crons.

Only release this, when the feature itself was released.

PR for the feature: https://github.com/getsentry/sentry-python/pull/1967

### CODE REVIEW

File: src/platform-includes/crons/setup/python.celery.mdx [hunk 1]
```diff
@@ -1,46 +1,92 @@
...
-        integrations=[CeleryIntegration()],
+        integrations=[CeleryIntegration(celery_app=app)],  # 👈 pass Celery app.
         environment="local.dev.grace",
-        release="v1.0.7-a1",
+        release="v1.0",
     )
 ```
 
-Finally, link your Celery task to a Sentry Cron Monitor:
+This will now auto discover all the tasks in your Celery Beat schedule and will capture telemetry data when the task is started, when it finishes or fails.
```
Comment (lizokm) [inline src/platform-includes/crons/setup/python.celery.mdx:44]:
```suggestion
Once Sentry Crons is set up, tasks in your Celery Beat schedule will be auto-discoverable, and telemetry data will be captured when a task is started, when it finishes, and when it fails. 

```

File: src/platform-includes/crons/setup/python.celery.mdx [hunk 2]
```diff
@@ -1,46 +1,92 @@
...
+
+<Note>
+
+You **do not need to create Cron Monitors** for your tasks on Sentry.io, we will do this for you.
+
+</Note>
+
+Start your Celery Beat and workers and see your tasks being monitored at [https://sentry.io/crons/](https://sentry.io/crons/).
+
```
Comment (lizokm) [inline src/platform-includes/crons/setup/python.celery.mdx:53]:
```suggestion
Start your Celery Beat and workers and see your tasks being monitored at [https://sentry.io/crons/](https://sentry.io/crons/).

<Note>

You don't need to create Cron Monitors for your tasks on Sentry.io, we'll do it for you.

</Note>

```
It makes for a smoother user experience to have the note at the end :)

File: src/platform-includes/crons/setup/python.celery.mdx [hunk 3]
```diff
@@ -1,46 +1,92 @@
...
+We provide a lightweight decorator to make it easy to monitor individual tasks.
+
+Add the `@sentry_sdk.monitor` decorator to your Celery task (or really just any function) and supply a `monitor_slug` of a monitor created previously on Sentry.io.
+Every time the task (or function) is now executed telemetry data will be captured when the task is started, when it finishes or fails.
+
+<Note>
+
+Make sure the Sentry `@sentry_sdk.monitor` decorator is **below** Celery's `@app.task` decorator.
+
+</Note>
```
Comment (lizokm) [inline src/platform-includes/crons/setup/python.celery.mdx:65]:
```suggestion
## Manual Task Monitoring

We provide a lightweight decorator to make monitoring individual tasks easier. To use it, add `@sentry_sdk.monitor` to your Celery task (or any function), then supply a `monitor_slug` of a monitor created previously on Sentry.io. Once this is done, every time the task (or function) is executed, telemetry data will be captured when a task is started, when it finishes, and when it fails.

<Note>

Make sure the Sentry `@sentry_sdk.monitor` decorator is below Celery's `@app.task` decorator.

</Note>
```

File: src/platform-includes/crons/setup/python.mdx [hunk 4]
```diff
@@ -1,12 +1,18 @@
 ## Job Monitoring
 
-Use the Python SDK to monitor and notify you if your job is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
+Use the Python SDK to monitor and notify you if your periodic task is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
+
+<Alert level="info" title="Note on Celery">
+
+If you are using **Celery Beat** to run you periodic tasks, have a look at our [Celery Beat Auto Discovery](/platforms/python/guides/celery/crons/)
+
+</Alert>
```
Comment (lizokm) [inline src/platform-includes/crons/setup/python.mdx:9]:
```suggestion
<Alert level="info" title="Note on Celery">

If you're using **Celery Beat** to run you periodic tasks, have a look at our [Celery Beat Auto Discovery documentation](/platforms/python/guides/celery/crons/).

</Alert>
```

**Review by lizokm — APPROVED** [review]
A few small tweaks, otherwise looks good.

File: src/platform-includes/crons/setup/python.celery.mdx (hunk 1 above)
Comment (shanamatthews) [reply src/platform-includes/crons/setup/python.celery.mdx:44]: I edited this suggestion to lowercase "beat". AFAICT the extension is called "Celery Beat" but it refers to its worker and beat services with lowercase.

File: src/platform-includes/crons/setup/python.celery.mdx (hunk 2 above)
Comment (shanamatthews) [reply src/platform-includes/crons/setup/python.celery.mdx:53]: edited your suggestion: "Celery Beat and workers" -> "Celery beat and worker services"

File: src/platform-includes/crons/setup/python.mdx (hunk 4 above)
Comment (shanamatthews) [reply src/platform-includes/crons/setup/python.mdx:9]: edited your suggestion to fix a typo ("you" -> "your")

**Review by shanamatthews — APPROVED** [review]

**Pull request merged**